### PR TITLE
docs(readme): Added development process

### LINKED ISSUE #5: [Docs]: Add Development Process to README
### Is your feature request related to a problem? Please describe.

README does not have Development Process. 

### Describe the solution you'd like.

Add the Development Process description to the README document.

### Describe alternatives you've considered

No alternatives accepted.

### Additional context

N/A

### PR DESCRIPTION
**Pull request by jacky-p**
## Changes
1. Added Development Process to README

## Purpose
It adds an explanation of the Development Process for the project.  

## Learning
[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
-used as a resource on how to format README

Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +39,11 @@ Step-by-step instructions on how to get the development environment running.
 
 Instructions and examples for using the project. Include code snippets and screenshots as needed.
 
+## Development Process
+
+* Create feature branch from `main`
+* Implement changes and commit frequently
+* Open PR into `main` when ready
```
Comment (elimanzo) [inline README.md:46]: Maybe add a  link [here](https://github.com/BizzNEST/Standards-and-Practices/tree/main/standards) that states follow bizzNest GitHub Branching/Commit/PR Standards

**Review by elimanzo — CHANGES_REQUESTED** [review]
Looks good just 1 Small nit

**Review by elimanzo — APPROVED** [review]
LGTM!

**Review by lilianaconstantino — APPROVED** [review]
Looks good.

**Pull request merged**